qiskit/transpiler/passes/scheduling/base_scheduler.py

```diff
@@ -168,7 +168,7 @@ class BaseSchedulerTransform(TransformationPass):
         it may separately schedule qubit and classical register,
         insertion of the delay yields unnecessary longer total execution time.
 
-        .. parsed-literal::
+        .. code-block:: text
             : Quantum Circuit, first-xgate
             0 ░▒▒▒░░░░░░░░░░░░░░░
             1 ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
```

Comment:
If the newlines matter, here's another file where they're missing.  If they do matter, maybe it's an indication that this file is missing from the docs?